assignments/array-methods.js

```diff
@@ -58,8 +58,8 @@ const runners = [
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs. Combine both the first and last names and populate a new array called `fullNames`. This array will contain just strings.
 const fullNames = [];
-runners.forEach(function(runners){
-    return fullNames.push(`${runners.first_name} ${runners.last_name}`);
+runners.forEach(function(runner, index){
+    return fullNames.push(`${runner.first_name} ${runner.last_name}`);
 });
 console.log(fullNames);
 
```

Comment:
no need for a return with the forEach loop.  a forEach doesn't return anything it just loops over an object and performs some kind of action on each item in the array.  ``` .map() ``` is the array method that loops over an array, performs an action on each item and then returns a new array and does not modify the original.